test/WebJobs.Extensions.CosmosDBCassandra.Tests/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions.Tests.Common;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Security;
+using System.Reflection;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+
+namespace Microsoft.Azure.WebJobs.Extensions.CosmosDBCassandra.Tests.Trigger
+{
+    public class CosmosDBTriggerAttributeBindingProviderTests
```

Comment:
The idea of this set of tests is to verify behavior when the user creates a Function with different combinations of parameters.

We create different combinations of scenarios, expose them as statics, like:

`public static IEnumerable<object[]> ValidCosmosDBTriggerBindingsWithAppSettingsParameters`

that leverages combinations of parameters for that scenario.

So if you want to test how the Trigger behaves if the ContactPoint is the name of a setting that exists in the Environment, you can do so, or on the contrary, you test how the Trigger behaves if the user does not pass all parameters, or passes an invalid one, etc.